src/frontend/src/icp/utils/ic.utils.ts

```diff
@@ -0,0 +1,4 @@
+import { ICP_TOKEN_ID } from '$env/tokens.env';
+import type { TokenId } from '$lib/types/token';
+
+export const isSupportedIcTokenId = (tokenId: TokenId): boolean => ICP_TOKEN_ID === tokenId;
```

Comment:
You can remove this. Generally speaking we should already have enough selector in place.

Reply:
Cool! I will look them up, where to find them